### DISCUSSION TITLE
Welcome to compose Discussions!

### DISCUSSION THREAD

**Discussion by onweru** [onweru/compose#34] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by varghesejose2020** [comment]
@onweru  please help 
E:\MyNewSite\themes\compose\exampleSite>hugo server
Error: we found a go.mod file in your project, but you need to install Go to use it. See https://golang.org/dl/.: module "github.com/onweru/compose" not found; either add it as a Hugo Module or store it in "E:\\hastham\\themes\\compose\\exampleSite\\themes".: module does not exist

**Comment by onweru** [comment]
Hi @varghesejose2020,

1. Ensure you have Hugo extended 
2. Install go in your path. It’s a small binary 

Which OS are you running sir?

**Reply by varghesejose2020** [reply]
@onweru 1. Hugo Static Site Generator v0.79.0/extended windows/amd64 BuildDate: unknown
OS:windows 8.1

**Reply by onweru** [reply]
That’s good. Consider installing go https://golang.org/

**Reply by onweru** [reply]
Alternatively, you can delete these files: [go.sum](https://github.com/onweru/compose/blob/master/exampleSite/go.sum) and [go.mod](https://github.com/onweru/compose/blob/master/exampleSite/go.mod). Then ensure you’re not including compose theme as a Hugo module. See the attached tool tip [from the docs](https://docs.neuralvibes.com/docs/compose/install-theme/)

![57C4489F-FB29-4F85-AE5B-4A854834A19E](https://user-images.githubusercontent.com/16350351/110766049-d44c2580-8265-11eb-9ffc-ec80848bdb12.jpeg)

**Reply by varghesejose2020** [reply]
installing **go** is not important?

**Reply by varghesejose2020** [reply]
replace the code in the config.toml with mentioned code right?

**Reply by onweru** [reply]
> installing **go** is not important?

It’s only one of the options.

**Reply by onweru** [reply]
> replace the code in the config.toml with mentioned code right?

Yes

**Reply by varghesejose2020** [reply]
i will install go and check it work or not

**Comment by varghesejose2020** [comment]
I installed Go in my programfile and run hugo server but same error exists

**Comment by varghesejose2020** [comment]
thank you worked :+1:  I restart the system

**Comment by varghesejose2020** [comment]
how to edit the content
